_includes/alumni.html

```diff
@@ -16,61 +16,74 @@ <h2 class="separate" style="color: ;">Alumni</h2>
-            <dt>  Ryan Cheney</dt><dd><pre>  </pre></dd>
-            <dt>  Samantha Hauserman-</dt><dd> Masters Student at University of Arizona</dd>
-            <dt>Sanjana Vasudevan-</dt><dd> Software Developer</dd>
-            <dt> SatyaSwaroop Boddu- </dt><dd> Software Engineer at EPIC Systems</dd>
-            <dt>  Sayalee Mehendale-</dt><dd> Software Engineer at Cisco Systems</dd>
-            <dt> Bianca Zietal- </dt><dd> Researcher (Alumna)</dd>
-            <dt>  Simbarashe Sithole</dt><dd><pre>  </pre></dd>
-            <dt> Sowjanya Ambati- </dt><dd> Software Developer (Alumna)</dd>
-            <dt>Supraj Kolluri-</dt><dd>Amazon</dd>
-            <dt>  Sushmita Kistamsetty</dt><dd><pre>  </pre></dd>
-            <dt>  Tushar Jain</dt><dd><pre>  </pre></dd>
-            <dt>  Veena Borannagowda-</dt><dd> Software Developer (Alumna)</dd>
-            <dt> Viraaj Navalekar-</dt><dd>Software Engineer at Facebook </dd>
-            <dt> Yogananda Kishore Yalugoti- </dt><dd> Amazon</dd>              
-            <dt> Zachary Rudebeck</dt>
-          </dl>
-          </div>
+			<dt>  Neehaarika Dasari</dt><dd><pre>  </pre></dd>
+			<dt>  Nischal Samji</dt><dd><pre>  </pre></dd>
+			<dt>  Prajakta Samant-</dt><dd> Software Developer Engineer at Microsoft</dd>
+			<dt>  Pranav Sharan- </dt><dd> Software Developer</dd>
+			<dt>  Raghav Kukreja</dt><dd><pre>  </pre></dd>
+			<dt>  Ramki Subramanian-</dt><dd> Senior Software Engineer at Echostar Corporation</dd>
+			<dt>  Ram Kumar Kumaresan-</dt><dd> Software Engineer at PayPal</dd>
+			<dt>  Renil Joseph</dt><dd><pre>  </pre></dd>
+			<dt>  Rohit Pendbhaje-</dt><dd> Software Developer at Capgemini</dd>
+			<dt>  Ryan Cheney</dt><dd><pre>  </pre></dd>
+			<dt>  Samantha Hauserman-</dt><dd> Masters Student at University of Arizona</dd>
+			<dt>  Sanjana Vasudevan-</dt><dd> Software Developer</d>
+			<dt>  SatyaSwaroop Boddu- </dt><dd> Software Engineer at EPIC Systems</dd>
+			<dt>  Sayalee Mehendale-</dt><dd> Software Engineer at Cisco Systems</dd>
+			<dt>  Bianca Zietal- </dt><dd> Researcher (Alumna)</dd>
+			<dt>  Simbarashe Sithole</dt><dd><pre>  </pre></dd>
+			<dt>  Sowjanya Ambati- </dt><dd> Software Developer (Alumna)</dd>
+			<dt>  Supraj Kolluri-</dt><dd>Amazon</dd>
+			<dt>  Sushmita Kistamsetty</dt><dd><pre>  </pre></dd>
+			<dt>  Tushar Jain</dt><dd><pre>  </pre></dd>
+			<dt>  Veena Borannagowda-</dt><dd> Software Developer (Alumna)</dd>
+			<dt> Viraaj Navalekar-</dt><dd>Software Engineer at Facebook </dd>
+			<dt> Yogananda Kishore Yalugoti- </dt><dd> Amazon</dd>  
+			<dt>  Namratha Olety Venkatesh</dt><dd><pre>  </pre></dd>
+			<dt>  Nayely Velez-Cruz</dt><dd><pre>  </pre></dd>
+			<dt>  Viraaj Navalekar-</dt><dd>Software Engineer at Facebook </dd>
+			<dt>  Yogananda Kishore Yalugoti-</dt><dd> Amazon</dd>              
+			<dt>  Zachary Rudebeck</dt><dd><pre>  </pre></dd>
+			<dt>  Ian Lobo<dt><dd><pre>  </pre></dd>
+			<dt>  Megan Bromley<dt><dd><pre>  </pre></dd>
```

Comment:
it looks like Ian and Megan are mentioned twice now? also, please put them in the right alphabetical order.